content/en/s3-to-snowflake/docs/create-pipeline.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Create the Pipeline
+weight: 3
+---
+
+{{% alert title="Make sure you have a Mezmo account" %}}
+
+In order to complete this workshop, you will need a Mezmo account with Pipeline enabled.  Note that this as a technical preview, Pipeline may not be enabled off the bat.  If you do not have an account, you can sign up for a free trial [here](https://app.mezmo.com/signup) and/or if you don't have Pipeline you can reach out to us at [pipeline@mezmo.com](mailto:pipeline@mezmo.com) to get it enabled.
```

Comment:
Should be `is a` vs `as a`?

Reply:
Also, instead of `pipeline@mezmo.com`, it should be `workshop-support@mezmo.com`.